### ISSUE TITLE
Goof stahp

### ISSUE THREAD

**Issue by Lumipharon** [tgstation/-tg-station#8325]
When you mix the reagent Life, it creates a reaction, and leaves you with 3u of (as far as I can tell) completely useless and inert Life reagent.

This is the exact same thing that led to the LDM server crash thing, so I can only assume all goon reactions have this issue.

Does your old PR fix this?

**Comment by Iamgoofball** [comment]
>exact same thing

That's not the same thing. 

Ldm was having problems with smoke and it was LDM specific.

**Comment by Dorsisdwarf** [comment]
Lum go test whether this lets you make life smoke.

FILL THE HALLS WITH BEARS

**Comment by Iamgoofball** [comment]
It does not, the mob is only created upon reaction.

**Comment by Lumipharon** [comment]
goof, it's the sam base issue.
IE: the chem causes a reaction, but leaves behind the reagent for no reason.

**Comment by Iamgoofball** [comment]
@Lumipharon no, it isn't

the problem with LDM was with it reacting with literally everything it touched in smoke, causing crashes

this does not have that problem

**Comment by Lumipharon** [comment]
... And as I said, it only happened because the reaction left the reagent itself behind, which is the same thing.

When I discovered the LDM bug, I tested it on MSO's server. It ONLY crashed the server when you used the actual reagent (IE: potassium, phosphorus, sugar an LDM, as opposed to stable plasma, radium and whatever the 3rd chem is). when you used the ingredients for the reagent, it lagged a bit, but never caused a crash.

But this is beside the point. All the goon reactions afaik are causing a reaction an leaving behind the reagent itself when it shouldn't.
